shell/platform/windows/angle_surface_manager.cc

```diff
@@ -67,13 +67,10 @@ bool AngleSurfaceManager::InitializeEGL(
 }
 
 bool AngleSurfaceManager::Initialize() {
-  // TODO(dnfield): Enable MSAA here, see similar code in android_context_gl.cc
-  // Will need to plumb in argument from project bundle for sampling rate.
-  // https://github.com/flutter/flutter/issues/100392
   const EGLint config_attributes[] = {EGL_RED_SIZE,   8, EGL_GREEN_SIZE,   8,
                                       EGL_BLUE_SIZE,  8, EGL_ALPHA_SIZE,   8,
                                       EGL_DEPTH_SIZE, 8, EGL_STENCIL_SIZE, 8,
-                                      EGL_NONE};
+                                      EGL_SAMPLES,    4};
```

Comment:
We could configure this per backend but turning on MSAA for Skia might not be a bad thing to do? w/e TOOD